File: Desktop/application/backend/entity/Komentar.java
package com.example.demo.entity;


import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.sql.Date;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder



public class Komentar {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;


    @Column(
            nullable = false
    )
    private String ime;


    @Column(
            nullable = false
    )
    private String prezime;

    @Column(
            nullable = false
    )
    private String poruka;

    @Column(
            nullable = false
    )
    private int ocena;

    @Column(nullable = true)
    private Date datum;

}

File: Desktop/application/frontend/models/Komentar.ts
export class Komentar {
    id: number;
    ime: string;
    prezime: string;
    poruka: string;
    ocena: number;
    datum: Date;
  
    constructor() {
      this.id = 0;
      this.ime = '';
      this.prezime = '';
      this.poruka = '';
      this.ocena = 0;
      this.datum = new Date(); // Initialize with the current date
    }
  }
  